src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
```

Comment:
Consider using `const` instead of `let` for `fullCost` since it's not being reassigned after its initial calculation. This helps to prevent unintentional changes to the variable.